xcodeproj/internal/bazel_integration_files/generate_bazel_dependencies.sh

```diff
@@ -6,7 +6,7 @@ cd "$SRCROOT"
 
 # Calculate Bazel `--output_groups`
 
-readonly swift_outputs_regex='.*\.swiftdoc$|.*\.swiftmodule$|.*\.swiftsourceinfo$'
+readonly base_outputs_regex='.*\.a|.*\.swiftdoc$|.*\.swiftmodule$|.*\.swiftsourceinfo$'
```

Comment:
Looks like it is missing a `$` sign, but even after adding it the .a files still didn't get downloaded for me.